BitwardenShared/Core/Platform/Models/Domain/EnvironmentUrlData.swift

```diff
@@ -82,9 +82,24 @@ extension EnvironmentUrlData {
         subpageUrl(additionalPath: "recover-2fa")
     }
 
+    /// Gets the region depending on the base url.
+    var region: RegionType {
+        switch base {
+        case EnvironmentUrlData.defaultUS.base:
+            .unitedStates
+        case EnvironmentUrlData.defaultEU.base:
+            .europe
+        default:
+            .selfHosted
+        }
+    }
+
     /// The base url for send sharing.
     var sendShareURL: URL? {
-        subpageUrl(additionalPath: "send")
+        guard region != .unitedStates else {
+            return URL(string: "https://send.bitwarden.com/#")!
```

Comment:
🤔 When I create a send in the app with this URL, opening that URL in a web browser takes me to a vault login page instead of the send. Is that expected?

Reply:
Yep, you were right. On this particular case there was a "/" that shouldn't be there. Fixed now.